config/database.yml

```diff
@@ -80,6 +80,7 @@ test:
 #
 production:
   <<: *default
-  database: bill_buck_generator_production
-  username: bill_buck_generator
-  password: <%= ENV['BILL_BUCK_GENERATOR_DATABASE_PASSWORD'] %>
+  host: <%= ENV['DB_PORT_5432_TCP_ADDR'] %>
```

Comment:
I am slightly concerned that this will break heroku, but I'm up for trying it :D